include/digitalPins.h

```diff
@@ -0,0 +1,39 @@
+#ifndef __DIGITAL_H__
+#define __DIGITAL_H__
+
+#include "ina281.h"
+#include "PID.h"
+
+//Define macros for the digital pins
```

Comment:
Probably don't need this command anymore. I think this was more of a "TODO" type comment.